### ISSUE TITLE
Cut 3.3.0 Release Candidate

### ISSUE THREAD

**Issue by christopher-henderson** [zmap/zlint#630]
We have the following changelog since May. Most notably #629 fixed #626 which is a blocker for anyone attempting to use ZLint as a library, so I reckon that's a good point to cut a new release.

1b89405 bump zcrypto to v0.0.0-20210811211718-6f9bc4aff20f (#629)
9da3c9f disallow duplicate entries in config.json (#616)
4940d55 Test certificate generator doesn't create a certificate chain (#622)
e274215 split lint_sub_ca_aia_missing lint into an error lint for before CABF_BR 1.7.1 and a warning for after.  Add test data (#613)
7bba362 Code clarification to match BRs wording. (#621)
48b300e Update README.md (#614)
8e8930e subsitute the initialize method for a constructor in the Lint struct (#607)
dbd9bfd dep: upgrade to latest ZCrypto with permissive asn1 parsing (#611)

**Comment by zakird** [comment]
Makes sense to me. Are #627, #619, #609 apt to close here soon such that we could include them? They don't seem large in scope, but there's been a lot of discussion going on.

**Comment by christopher-henderson** [comment]
I went ahead and cut the RC1 because the library compilation issue was enough of a blocker for those consumers that I felt that getting out there was a good idea. I'll have to revisit those three, but if I recall correctly they're in the "it's complicated" limbo of upstream policy decisions.

**Comment by christopher-henderson** [comment]
#619 at the least is largely dependent upon #624 in order for the lint to be deterministic.
